Makefile

```diff
@@ -460,3 +477,6 @@ clean:
 	@$(MAKE) --no-print-directory -C test clean
 	@echo "Uninstalling OSDK"
 	rm -f $(CARGO_OSDK)
+
+Clean: clean
```

Comment:
Let's not name this "Clean" it's way to similar to "clean". People will straight up not notice the difference. I think I remember people calling this "clean_all" back in the day. I just want it clearly different though.